Warning: '/work/ssahibul_umass_edu/.conda/envs' already in 'envs_dirs' list, moving to the bottom
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [96,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [97,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [98,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [99,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [100,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [101,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [102,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [103,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [104,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [105,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [106,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [107,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [108,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [109,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [110,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [111,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [112,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [113,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [114,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [115,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [116,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [117,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [118,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [119,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [120,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [121,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [122,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [123,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [124,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [125,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [126,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [55,0,0], thread: [127,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [0,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [1,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [2,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [3,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [4,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [5,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [6,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [7,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [8,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [9,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [10,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [11,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [12,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [13,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [14,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [15,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [16,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [17,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [18,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [19,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [20,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [21,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [22,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [23,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [24,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [25,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [26,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [27,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [28,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [29,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [30,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/cuda/ScatterGatherKernel.cu:144: operator(): block: [7,0,0], thread: [31,0,0] Assertion `idx_dim >= 0 && idx_dim < index_size && "index out of bounds"` failed.
Downloading https://github.com/kimiyoung/planetoid/raw/master/data/ind.cora.x
Downloading https://github.com/kimiyoung/planetoid/raw/master/data/ind.cora.tx
Downloading https://github.com/kimiyoung/planetoid/raw/master/data/ind.cora.allx
Downloading https://github.com/kimiyoung/planetoid/raw/master/data/ind.cora.y
Downloading https://github.com/kimiyoung/planetoid/raw/master/data/ind.cora.ty
Downloading https://github.com/kimiyoung/planetoid/raw/master/data/ind.cora.ally
Downloading https://github.com/kimiyoung/planetoid/raw/master/data/ind.cora.graph
Downloading https://github.com/kimiyoung/planetoid/raw/master/data/ind.cora.test.index
Processing...
Done!
Traceback (most recent call last):
  File "testing2layer.py", line 34, in <module>
    netTest2layer(argv) 
  File "/work/ssahibul_umass_edu/broadcaster_nodes/spring23/layer2Test.py", line 79, in netTest2layer
    pred = model(data).argmax(dim=1)
  File "/work/ssahibul_umass_edu/.conda/envs/bc38/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/work/ssahibul_umass_edu/broadcaster_nodes/spring23/broadcaster_nets_2_3.py", line 123, in forward
    x = self.layers[layer_num].forward(x,edge_index,bc_features,bc_assigment)
  File "/work/ssahibul_umass_edu/broadcaster_nodes/spring23/BCMPLayer.py", line 94, in forward
    Za1 = self.baselayer3(xZ,bc_assigment_edges)
  File "/work/ssahibul_umass_edu/.conda/envs/bc38/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/ssahibul_umass_edu/.local/lib/python3.8/site-packages/torch_geometric/nn/conv/sage_conv.py", line 131, in forward
    out = self.propagate(edge_index, x=x, size=size)
  File "/home/ssahibul_umass_edu/.local/lib/python3.8/site-packages/torch_geometric/nn/conv/message_passing.py", line 391, in propagate
    out = self.aggregate(out, **aggr_kwargs)
  File "/home/ssahibul_umass_edu/.local/lib/python3.8/site-packages/torch_geometric/nn/conv/message_passing.py", line 514, in aggregate
    return self.aggr_module(inputs, index, ptr=ptr, dim_size=dim_size,
  File "/home/ssahibul_umass_edu/.local/lib/python3.8/site-packages/torch_geometric/nn/aggr/base.py", line 109, in __call__
    elif index.numel() > 0 and dim_size <= int(index.max()):
RuntimeError: CUDA error: device-side assert triggered
CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
Faiss assertion 'err == cudaSuccess' failed in virtual void faiss::gpu::StandardGpuResourcesImpl::deallocMemory(int, void*) at /home/conda/feedstock_root/build_artifacts/faiss-split_1644327811086/work/faiss/gpu/StandardGpuResources.cpp:513; details: Failed to cudaFree pointer 0x7f51c4000000 (error 710 device-side assert triggered)
/var/spool/slurm/slurmd/job6225799/slurm_script: line 46: 1429186 Aborted                 python testing2layer.py ${9}br2_${4}_${1}_test${3}_iter${2}_clus${5}_wait${6}_lr${7}_lrdecay${8}_$SLURM_JOB_ID $1 $2 $3 $4 $5 $6 $7 $8
